recipes/libgd/all/patches/2.3.2-use-cci.patch

```diff
@@ -11,6 +11,16 @@ index 57cd95d..11a8409 100644
  	ENDIF (ENABLE_WEBP)
  
  	IF (ENABLE_HEIF)
+-		FIND_PACKAGE(HEIF REQUIRED)
++		FIND_PACKAGE(libheif REQUIRED)
```

Comment:
Could do `deps.set_property("libheif", "cmake_file_name", "HEIF")` instead to keep the patch a bit simpler.